packages/cli/src/utils/typescript.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path'
+import { existsSync } from 'fs'
+import chalk from 'chalk'
+import consola from 'consola'
+import { warningBox } from './formatting'
+
+export async function getTypescriptConfig(rootDir, extraOptions = {}) {
+  const tsConfigPath = path.resolve(rootDir, 'tsconfig.json')
+
+  if (!existsSync(tsConfigPath)) {
+    return undefined
+  }
+
+  consola.info('TypeScript environment detected')
+
+  try {
+    const { setup } = require('@nuxt/typescript')
+    await setup(tsConfigPath, { transpileOnly: !!extraOptions._start })
+  } catch (e) {
+    if (e.code === 'MODULE_NOT_FOUND') {
+      const messageLines = [
```

Comment:
Can we use template string here?

Reply:
What do you mean by template string ? :)

Reply:
`Please install...

${chalk.bold(..)}
...
`
and so on

Reply:
Yep. Just a code cleanup suggestion

Reply:
Oh yeah alright, sure !